DemoApp/src/Argument.h

```diff
@@ -56,6 +57,19 @@ typedef struct Argument {
         }
     }
 
+    std::string typeString() {
+        switch (type) {
+            case ArgumentType::INTEGER:
+                return "Integer";
+            case ArgumentType::REAL:
+                return "Real";
+            case ArgumentType::STRING:
+                return "String";
+            default:
+                return "";
+        }
+    }
+
 } Argument;
 
 #endif  // DEMOAPP_ARGUMENT_H
```

Comment:
newline :P

Reply:
Not my changes, it was'n there even before this PR. Its your code btw